Makefile.pre.in

```diff
@@ -244,7 +244,7 @@ LIBOBJS=	@LIBOBJS@
 PYTHON=		python$(EXE)
 BUILDPYTHON=	python$(BUILDEXE)
 
-PYTHON_FOR_REGEN=@PYTHON_FOR_REGEN@
+PYTHON_FOR_REGEN?=@PYTHON_FOR_REGEN@
```

Comment:
Just out of curiosity, what does the `@...@` notation do?

Reply:
It substitutes the values that autotools calculate in the configure step. Basically this file is a template that will be used to generate the real makefile after configure analyses the system.

Reply:
Thanks for the explanation!